src/medCore/process/medAbstractProcess.h

```diff
@@ -21,6 +21,17 @@
 
 class medAbstractProcessPrivate;
 
+namespace DATAERROR
+{
+enum DATAERROR
+{
+    PIXELTYPE = 2, //! Pixel type not yet implemented
+    DIMENSION = 3, //! Not a 3D volume
+    MESHTYPE  = 4, //! Not a mesh
+    UNDEFINED = 5, //! Miscellanous
```

Comment:
- DATAERROR is a type and should be named like other types: DataError.
- when capitalising enum values, use underscore as a separator: PIXEL_TYPE
- Number the first enum value and leave the others unnumbered. The compiler will automatically assign successive values to them, and it avoids errors if someone adds one in between.